### PR TITLE
feat: node revamp milestone 1

### PR DESCRIPTION
**Pull request by 01zulfi**
## Because

node revamp

## This PR

node revamps

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [ ] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [ ] The title of this PR follows the `keyword: brief description of change` format, using one of the following keywords:
    - `Feature` - adds new or amends existing user-facing behavior
    - `Chore` - changes that have no user-facing value, refactors, dependency bumps, etc
    - `Fix` - bug fixes
-   [ ] The `Because` section summarizes the reason for this PR
-   [ ] The `This PR` section has a bullet point list describing the changes in this PR
-   [ ] I have verified all tests and linters pass after making these changes.
-   [ ] If this PR addresses an open issue, it is linked in the `Issue` section
-   [ ] If applicable, this PR includes new or updated automated tests

### CODE REVIEW

**Comment by MaoShizhong** [comment]
Just a heads up that the new env lesson will need adding here too :tada:

File: db/fixtures/paths/full_stack_javascript/courses/databases.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  course.show_on_homepage = true
+  course.badge_uri = 'badge-database.svg'
+end
+
+# +++++++++++++++++++
+# SECTION - Databases
+# +++++++++++++++++++
+course.add_section do |section|
+  section.title = 'Databases'
+  section.description = 'ORMS in Node.js does a lot of the heavy lifting with connecting and querying a database but there will be times you need to tweak a query to the database using raw SQL. Learning how to query efficiently will help your understanding of what Rails helps abstract away.'
```
Comment (MaoShizhong) [inline db/fixtures/paths/full_stack_javascript/courses/databases.rb:17]: Nit
```suggestion
  section.description = 'ORMs in Node.js do a lot of the heavy lifting with connecting and querying a database but there will be times you need to tweak a query to the database using raw SQL. Learning how to query efficiently will help your understanding of what Rails helps abstract away.'
```

File: db/fixtures/paths/full_stack_javascript/courses/node_js.rb [hunk 2]
```diff
@@ -24,39 +24,28 @@
...
 # +++++++++++++++++++++++++++
-# SECTION - Express & Mongoose
+# SECTION - Express
 # +++++++++++++++++++++++++++
 course.add_section do |section|
-  section.title = 'Express & Mongoose'
-  section.description = 'Here we finally get to Express, the most popular back-end JavaScript framework, and learn to build applications with MongoDB using Mongoose.'
-  section.identifier_uuid = '667788c1-67b9-4ed1-9b72-c5e8bc6f9009'
+  section.title = 'Express'
+  section.description = 'Here we finally get to Express, the most popular back-end JavaScript framework, and learn to build applications with PostgreSQL database.'
```
Comment (MaoShizhong) [inline db/fixtures/paths/full_stack_javascript/courses/node_js.rb:36]: Nit
```suggestion
  section.description = 'Here we finally get to Express, the most popular back-end JavaScript framework, and learn to build applications with a PostgreSQL database.'
```

File: db/fixtures/lessons/node_js_lessons.rb [hunk 3]
```diff
@@ -30,88 +30,109 @@ def node_js_lessons
...
-      description: 'This lesson shows you how to set up view templates.',
+    'Forms and Data Handling' => {
+      title: 'Forms and Data Handling',
+      description: 'How to use forms and its data validation in Express',
       is_project: false,
-      github_path: '/nodeJS/express_and_mongoose/express_104_view_templates.md',
-      identifier_uuid: '3a3cddd5-e2cf-4545-bc3d-60c692018231',
+      github_path: '/nodeJS/express/forms_and_data_handling.md',
+      identifier_uuid: '2db51796-85a1-49d2-a40e-3f94e93b5dd1',
     },
```
Comment (MaoShizhong) [inline db/fixtures/lessons/node_js_lessons.rb:97]: The lessons are applied in `node_js.rb` as
`Forms and Data Handling` -> `Installing PostgreSQL` -> `Using PostgreSQL`

Even if functionally it makes no difference, probably best to have the lessons in the same order in this file for consistency.

File: db/fixtures/paths/full_stack_javascript/courses/node_js.rb
Comment (MaoShizhong) [inline db/fixtures/paths/full_stack_javascript/courses/node_js.rb:1]: The course description on line 6 will need amending to reference PostgreSQL

**Review by MaoShizhong — CHANGES_REQUESTED** [review]

File: db/fixtures/paths/full_stack_javascript/courses/node_js.rb
Comment (01zulfi) [reply db/fixtures/paths/full_stack_javascript/courses/node_js.rb:1]: nice catch

**Review by KevinMulhern — APPROVED** [review]
Looks great to me @01zulfi! excited to see this go out 🔥

**Review by MaoShizhong — APPROVED** [review]
:rocket:

**Pull request merged**